app/src/main/java/io/github/sumukhshiv/memberquiz/GameActivity.java

```diff
@@ -0,0 +1,347 @@
+
+        //clear all button text when initializing
+        for (int i = 0; i < buttonArray.length; i++) {
+            buttonArray[i].setText("");
+        }
+        //ArrayList of all the names of images in the drawable folder
+        ArrayList<String> imageNames = new ArrayList<>();
+
+        //Names of all members from textfile
+        String[] members = {"Jessica Cherny", "Kevin Jiang", "Jared Gutierrez", "Kristin Ho", "Christine Munar", "Mudit Mittal", "Richard Hu", "Shaan Appel", "Edward Liu", "Wilbur Shi", "Young Lin", "Abhinav Koppu", "Abhishek Mangla", "Akkshay Khoslaa", "Andy Wang", "Aneesh Jindal", "Anisha Salunkhe", "Ashwin Vaidyanathan", "Cody Hsieh", "Justin Kim", "Krishnan Rajiyah", "Lisa Lee", "Peter Schafhalter", "Sahil Lamba", "Sirjan Kafle", "Tarun Khasnavis", "Billy Lu", "Aayush Tyagi", "Ben Goldberg", "Candice Ye", "Eliot Han", "Emaan Hariri", "Jessica Chen", "Katharine Jiang", "Kedar Thakkar", "Leon Kwak", "Mohit Katyal", "Rochelle Shen", "Sayan Paul", "Sharie Wang", "Shreya Reddy", "Shubham Goenka", "Victor Sun", "Vidya Ravikumar"};
+
+        //Go through drawable folder and add names of images to imageNames ArrayList
+        //CONTAINS SOME HARDCODED ELEMENTS
+        Field[] drawables = io.github.sumukhshiv.memberquiz.R.drawable.class.getFields();
+        for (Field f : drawables) {
+            try {
+                String temp = "R.drawable." + f.getName();
+                if (!temp.contains("_") && (!temp.contains("$") && (!temp.contains("serialVersionUID")))) {
+                    imageNames.add(f.getName());
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+
+
+        //Random number for Picture and correct name
+        Random rand = new Random();
+        int correctAnswerRandomNumber = rand.nextInt(44);
+        String displayImageName = imageNames.get(correctAnswerRandomNumber);
+
+        //Random number for button 1st incorrect
+        Random rand2 = new Random();
+        int wrong1 = rand.nextInt(44);
+        int wrong2 = rand.nextInt(44);
+        int wrong3 = rand.nextInt(44);
+
+        //Generate random numbers that are all unique
+        while (wrong1 == correctAnswerRandomNumber) {
+            wrong1 = rand.nextInt(44);
+        }
+        while (wrong2 == correctAnswerRandomNumber || wrong2 == wrong1) {
+            wrong2 = rand.nextInt(44);
+        }
+        while (wrong3 == correctAnswerRandomNumber || wrong3 == wrong1 || wrong3 == wrong2) {
+            wrong3 = rand.nextInt(44);
+        }
+        //Get the actual string using random index integer
+        String wrong1Name = imageNames.get(wrong1);
+        String wrong2Name = imageNames.get(wrong2);
+        String wrong3Name = imageNames.get(wrong3);
+
+
+        //id to set the image in the imageView
+        Context c = getApplicationContext();
+        int id = c.getResources().getIdentifier("drawable/"+displayImageName, null, c.getPackageName());
```

Comment:
can probably just call getPackageName with no object, you are inside the context already